api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet id="patientflags-unique-flagname-20171115-1523" author="ssmusoke">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="patientflags_flag"></tableExists>
+        </preConditions>
+        <comment>Add a unique constraint to the name of the flag</comment>
+        <addUniqueConstraint tableName="patientflags_flag" columnNames="name"></addUniqueConstraint>
```

Comment:
Shouldn't you give these constraints names and then first check if a constraint with that name does not already exist?